parserator_web/views.py

```diff
@@ -14,11 +14,38 @@ class AddressParse(APIView):
     renderer_classes = [JSONRenderer]
 
     def get(self, request):
-        # TODO: Flesh out this method to parse an address string using the
-        # parse() method and return the parsed components to the frontend.
-        return Response({})
+        '''Process a request to this API endpoint and return data representing the results of processing.'''
+
+        # Grab the address from our request data
+        address = request.query_params.get('address');
+
+        try:
```

Comment:
When writing `try / except` blocks, it's more explicit and less error prone to include only the lines that could raise exceptions you want to handle in the `try` block. For example, if you had a typo in your response dictionary, it would be handled the same as a parse error, which can lead to some baffling debugging. How about moving the creation of a successful response dictionary [to an `else` block](https://docs.python.org/3/tutorial/errors.html#handling-exceptions)?

Reply:
Thanks! I've been deep in JS world lately so I had completely forgotten about `else` blocks. Great solution to an issue I hadn't considered. Just implemented this change alongside the API response changes above.